ui/src/app/pages/data/data-set/export-dataset-modal.tsx

```diff
@@ -101,7 +101,7 @@ export const ExportDatasetModal: (props: Props) => JSX.Element = fp.flow(
       kernelType,
       genomicsAnalysisTool,
       generateGenomicsAnalysisCode: hasWgs(),
-      notebookName,
+      notebookName: appendNotebookFileSuffix(notebookName),
```

Comment:
the 2 changes in this file are the fixes
* export dataset to new notebook (here)
* export dataset to existing notebook (L191)